src/Health/Router.ts

```diff
@@ -67,4 +70,16 @@ export class Router {
       logger.error({ error }, 'Failed to update ipfsHashFailures on health')
     }
   }
+
+  blockDownloadedConsumer = async (blockDownloaded: BlockDownloaded): Promise<void> => {
+    const logger = this.logger.child({ method: 'blockDownloadedConsumer' })
+
+    logger.trace({ blockDownloaded }, 'Block downloaded, removing associated transactions')
+    try {
+      const transactionIds = pluck('transactionId', blockDownloaded.poetBlockAnchors)
```

Comment:
Can the same be achieved with `blockDownloaded.poetBlockAnchors.map(_ => _.transactionId)`? Referencing fields by strings is not type-safe nor refactor/rename friendly.

Reply:
done. I didn't take the commit suggestion because I needed to remove the import as well